crates/symbolicator/src/sources.rs

```diff
@@ -124,39 +127,56 @@ pub struct FilesystemSourceConfig {
     pub files: CommonSourceConfig,
 }
 
+#[cached]
+fn aws_regions() -> Vec<aws_sdk_ec2::model::Region> {
```

Comment:
Maybe using something like `once_cell::Lazy` might be a better idea than a proc macro just for this.
Also this answers my question of why we are pulling in the `ec2` crate. Isn’t there a better way to do this?